domain/src/main/java/org/oppia/android/domain/topic/TopicListController.kt

```diff
@@ -365,10 +365,26 @@ class TopicListController @Inject constructor(
     } != null
   }
 
-  private fun getStartedChapterProgressList(storyProgress: StoryProgress): List<ChapterProgress> =
-    getSortedChapterProgressListByPlayState(
-      storyProgress, playState = ChapterPlayState.STARTED_NOT_COMPLETED
-    )
+  private fun getStartedChapterProgressList(storyProgress: StoryProgress): List<ChapterProgress> {
+    val startedNotCompletedChapterList =
+      getSortedChapterProgressListByPlayState(
+        storyProgress, playState = ChapterPlayState.STARTED_NOT_COMPLETED
```

Comment:
This can happen for those lessons which were left partially complete by the user before they updated the app to a version that supports checkpointing.